### PR TITLE
feat: add arbitrum support

### PR DESCRIPTION
**Pull request by DarkGhost7**
# Check list below
from #242 

You must configure the network 

You must add the following constants:
- in ./yearn/networks.py:
- [x] specify the chainid in the same way as the others.
- [x] specify a network label in `Networks.label`.

- in ./yearn/constants.py:
- [x] specify Treasury address. (if Yearn has a treasury contract deployed) This would be the rewards() address of any vault.
- [x] specify Multisig address. This would be the governance() address of any vault
- [x] specify Strategist Multisig address. (if Yearn has a SMS deployed) This would be the management() address of any vault.

- in ./yearn/prices/constants.py:
- [x] add weth address to `tokens_by_network`.
- [x] add usdc address to `tokens_by_network`.
- [x] add dai address to `tokens_by_network`.
- [x] configure `stablecoins` using popular + "safe" stablecoins on the chain.

- in ./yearn/multicall2.py:
- [x] specify the appropriate multicall2 contract address.

- in ./yearn/prices/chainlink.py:
- [x] specify any chainlink feeds available on the chain in `feeds`.
- [x] specify the chainlink registry address in `registries` if applicable, or specify None if not applicable.

- in ./yearn/prices/uniswap/v2.py:
- [x] specify the factory and router address for any important uni v2 forks on the chain.

- in ./yearn/utils.py:
- [ ] specify `BINARY_SEARCH_BARRIER` as 0 if all historical data is available. If historical data not available due to a network upgrade/etc, specify the first block of the available historical data.

- in ./yearn/middleware/middleware.py:
- [x] specify `BATCH_SIZE` of approx 1 day based on avg block times on the chain.

- in ./scripts/historical-exporter.py:
- [x] configure `end` to equal the date + hour of deployment for the first Yearn product on the chain.
- [x] configure `data_query`. See `mapping` in ./yearn/outputs/victoria/output_helper.py to figure out the metric to specify based on what type of product was the first deployed on the chain.

- in ./scripts/historical-treasury-exporter.py:
- [x] configure `end` to equal the date + hour of deployment for the Yearn Treasury on the chain.
- [ ] configure `data_query` using the existing networks as an example.

- in ./scripts/historical-sms-exporter.py:
- [x] configure `end` to equal the date + hour of deployment for the Strategist Multisig on the chain.
- [ ] configure `data_query` using the existing networks as an example.

You also need to set up containers for each exporter on the new chain. This is a little more complicated but you can use existing containers as an example:
- [x] define common envs for the chain
- [x] add one new service entry for forwards exporter
- [x] add one new service entry for historical exporter
- [x] add one new service entry for treasury exporter
- [x] add one new service entry for historical treasury exporter
- [x] add one new service entry for sms exporter
- [x] add one new service entry for historical sms exporter
- [x] add one new service entry for wallet exporter
- [x] add one new service entry for transactions exporter
- [x] adapt entrypoint.sh

There still may be some things I'm missing, so its important to test to make sure everything works!

Once you've handled everything above, type `make all && make logs-all` into your terminal at the project root. The exporters will start running in docker and any exceptions will show in your terminal. If there are no exceptions, and everything appears to be running smoothly, it's time to submit your PR!

### CODE REVIEW

File: scripts/historical_exporter.py [hunk 1]
```diff
@@ -24,6 +24,10 @@ def main():
         # end: yvUSDC vault January-20-2022 06:10:45 AM +-6 UTC
         end = datetime(2022, 1, 20, tzinfo=timezone.utc)
         data_query = 'yearn_vault{network="GNO"}'
+    elif Network(chain.id) == Network.Arbitrum:
+        # end: yvUSDC 4.3 Sep-05-2021 09:05:50 PM +UTC
+        end = datetime(2021, 9, 05, tzinfo=timezone.utc)
+        data_query = 'yearn_vault{network="ARRB"}'
```
Comment (DarkGhost7) [inline scripts/historical_exporter.py:30]: `network="ARRB"`
not 100% sure this is correct but its the network label it has below

File: services/dashboard/docker-compose.yml [hunk 2]
```diff
@@ -49,6 +49,12 @@ x-common-ftm-envs: &common-ftm-envs
   - EXPLORER: $FTM_EXPLORER
   - FTMSCAN_TOKEN:
 
+x-common-arbi-envs: &common-arbi-envs
```
Comment (DarkGhost7) [inline services/dashboard/docker-compose.yml:52]: added arbi exporters and reordered the whole file to look cleaner

**Comment by DarkGhost7** [comment]
Work is ready for review for #354

**Comment by BobTheBuidler** [comment]
lgtm

**Review by BobTheBuidler — APPROVED** [review]

**Pull request merged**